scripts/send-slack-notification-bot.py

```diff
@@ -78,6 +79,7 @@ def main():
     release_info = {
         "version": version,
         "jira_card_link": read_file_content(os.path.join(shared_dir, "jira_link")),
```

Comment:
Maybe we should be consistent here.
The Jira link file is without `.txt` suffix, so we should either add it to Jira or remove it from the Polarion